api/v1alpha3/azurecluster_conversion.go

```diff
@@ -58,6 +58,8 @@ func (src *AzureCluster) ConvertTo(dstRaw conversion.Hub) error { // nolint
 	dst.Spec.NetworkSpec.APIServerLB.FrontendIPsCount = restored.Spec.NetworkSpec.APIServerLB.FrontendIPsCount
 	dst.Spec.NetworkSpec.NodeOutboundLB = restored.Spec.NetworkSpec.NodeOutboundLB
 
```

Comment:
nit: please remove the space between line 59 and 61.

Reply:
Blank line removed.
Regarding the e2e test, I will work on that.